pandas/io/spss.py

```diff
@@ -0,0 +1,33 @@
+from typing import Union, Sequence
+from pandas.core.api import DataFrame
+
+
+def read_spss(path: str,
+              usecols: Union[str, Sequence[str], None] = None,
+              categorical: bool = True) -> DataFrame:
+    """
+    Load an SPSS file from the file path, returning a DataFrame.
+
+    .. versionadded 0.25.0
+
+    Parameters
+    ----------
+    path : string
```

Comment:
does this accept a pathlike? Union[str, Path] ?

Reply:
`pathlib.Path` should work everywhere a path string is expected, so IMO it is not really necessary to explicitly add this. But if you want I can of course add it (this requires an extra `import pathlib` though).

Reply:
I've implemented this change.